frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -45,8 +46,33 @@ local function cleanupSelectedText(text)
     return text
 end
 
+local CLEAR_HIGHLIGHT_SELECTION = true
+local NOT_CLEAR_HIGHLIGHT_SELECTION = false
```

Comment:
You still need these ?
It indeed makes your `self:onStopHighlightIndicator(NOT_CLEAR_HIGHLIGHT_SELECTION)` quite clear, but well, if we were to do that everytime we pass a true/false parameter, we would need to update a lot of our code :)
Just use true/false, add a comment after the call : `-- need_clear_selection=true` if you feel it helps understanding (or not, and we'll go see the function definition to know what it means, like we usually do everywhere :)

Reply:
remove these two constants.